target_x: [1,  1,  0, 0, 0]
target_y: [0, -1, -1, 0, 0]
target_z: [0,  0,  0, 0, 1.57]